src/libstd/thread/mod.rs

```diff
@@ -112,6 +112,25 @@
 //! will want to make use of some form of **interior mutability** through the
 //! [`Cell`] or [`RefCell`] types.
 //!
+//! ## Naming threads
+//!
+//! Threads are able to have associated names for identification purposes. For example, the thread
+//! name is used in panic messages. By default, spawned threads are unnamed. To specify a name for
+//! a thread, build the thread with [`Builder`] and pass the desired thread name to
+//! [`Builder::name`]. To retrieve the thread name from within the thread, use [`Thread::name`].
+//!
+//! ## Stack size
+//!
+//! The default stack size for spawned threads is 2 MiB, though this particular stack size is
+//! subject to change in the future. There are two ways to manually specify the stack size for
+//! spawned threads:
+//!
+//! * Build the thread with [`Builder`] and pass the desired stack size to [`Builder::stack_size`].
+//! * Set the `RUST_MIN_STACK` environment variable to an integer representing the desired stack
+//!   size.
```

Comment:
Please specify the unit of `RUST_MIN_STACK` is "bytes" (e.g. `RUST_MIN_STACK=4194304` for a 4 MiB stack).